src/bson.rs

```diff
@@ -1054,6 +1057,131 @@ impl Binary {
     }
 }
 
+#[cfg(feature = "uuid-0_8")]
+#[cfg_attr(docsrs, doc(cfg(feature = "uuid-0_8")))]
```

Comment:
typically we group all the attributes together, so this would be better below with the `enum` declaration. Also note that the doctest won't get run unless these are met, even if they come after it.